app/views/pages/security.html.erb

```diff
@@ -0,0 +1,211 @@
+  </p>
+
+  <p>
+    Curly's GPG key has key id 41C6E930 (full fingerprint
+    0145 FD2B 52E8 0A8E 329A 16C7 AC68 AC04 41C6 E930) and email address
+    richard@python.org
+  </p>
+
+  <h2>RubyGems Security Release Policy</h2>
+  <p>Security issues:</p>
+  <p>Severe Security Issues:</p>
+  <p>Unsupported Security Issues:</p>
+
+  <h2>Your Security</h2>
+  <h3>Securely updating your RubyGems Gem</h3>
+
+
+  <h3>Securely pushing Gems</h3>
+  <p>
+    You may <a href="http://guides.rubygems.org/security/#building-gems">
+      sign your gems per the guides</a>.
+  </p>
+
+  <h3>Securely download Gems</3>
+    <p>
+      The SHA256hash provided with files on RubyGems.org exists only to provide some
+      download corruption protection. It is not intended to provide any sort of
+      security regarding tampering. Please use GPG signing for that.
+    </p>
+
+    Also, see <a href="https://gist.github.com/bf4/4223e83e8becacfb2a8e">this gist</a>
+
+    <h2>Reporting RubyGems.org Website Problems</h2>
+    <p>
+      To report a bug or to suggest an enhancement,
+      send an e-mail to <webmaster@rubygems.org>.
+    </p>
+
+    <p>
+      If you're reporting a problem (a broken link, typo,
+      formatting glitch, etc),  please include the URL of the affected page.
+    </p>
+
+    <p>
+      Note that this address is only for problems with the main rubygems.org
+      website.
+    </p>
+
+    <p>
+      For problems with RubyGems.org please use the
+      <a href="https://github.com/rubygems/rubygems.org">RubyGems.org issue tracker</a>.
+    </p>
+
+    <h2>Disclosure Policy</h2>
+    <p>
+      RubyGems.org has a 5 step disclosure policy.
+    </p>
+
+    <ol>
+      <li>
+        Security report received and is assigned a primary handler. This person will coordinate the fix and release process.
+      </li>
+      <li>
+        Problem is confirmed and, a list of all affected versions is determined. Code is audited to find any potential similar problems.
+      </li>
+      <li>
+        Fixes are prepared for all releases which are still supported. These fixes are not committed to the public repository but rather held locally pending the announcement.
+      </li>
+      <li>
+        A suggested embargo date for this vulnerability is chosen and distros@openwall is notified. This notification will include patches for all versions still under support and a contact address for packagers who need advice back-porting patches to older versions.
+      </li>
+      <li>
+        On the embargo date, the RubyBems security mailing list is sent a copy of the announcement. The changes are pushed to the public repository and new gems released to rubygems. At least 6 hours after the mailing list is notified, a copy of the advisory will be published on
+        the <a href="http://blog.rubygems.org/">RubyGems.org blog</a>.
+      </li>
+    </ol>
+
+    <p>
+      Typically the embargo date will be set 72 hours from the time vendor-sec
```

Comment:
Thanks @reedloden for the review.  Don't spend too much time on any details, though, 'cause it'll be changing quickly.  I'll update with a comment when ready for feedback.